dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -94,6 +98,9 @@ http.remoteip.useRemoteIpInAccessLog.desc=If this property is set to true and th
 http.compression=Compression Options
 http.compression.desc=An element that is configured within the httpEndpoint element so that the associated HTTP channel can consider compressing response bodies based on the client's Accept-Encoding header.
 
+http.cookies=Cookie Options
+http.cookies.desc=An element that is configured within the httpEndpoint element so that the associated HTTP channel can consider cookie specific configurations.
```

Comment:
change
cookie specific
to
cookie-specific

Reply:
Changes added